src/funcade/middleware/buddy.clj

```diff
@@ -9,8 +9,11 @@
 (defn validate-scope [handler request required-scope]
   (let [decoded-token (request :identity)]
     (let [scopes (->> (decoded-token :scope)
-                      (map #(keyword %)))]
-      (if (some #{required-scope} scopes)
+                      (map #(keyword %)))
+          allowed-scopes (if (coll? required-scope)
```

Comment:
if it is possible for `required-scope` to be a sequence, let's call it `require-scopes`